Marlin/Marlin_main.cpp

```diff
@@ -7567,12 +7567,13 @@ inline void gcode_M42() {
 
 #endif // Z_MIN_PROBE_REPEATABILITY_TEST
 
-#if ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(UBL_G26_MESH_VALIDATION)
+#if ENABLED(UBL_G26_MESH_VALIDATION)
```

Comment:
I guess this will just be called `G26_MESH_VALIDATION` after this PR.

Reply:
Yeah...   Agreed...